.github/workflows/companion-deploy.yml

```
@@ -4,6 +4,8 @@ name: Companion Deploy
 on:
   push:
     branches: [main]
+    paths:
+    - 'packages/@uppy/companion/**'
```

Should we deploy only on Companion releases?
```suggestion
    - 'packages/@uppy/companion/CHANGELOG.md'
```

---

Even better!

---

but we don't do that anywhere else, we don't publish 'development versions' to the npm registry, so now that I think of it why would the docker version be the exception?

---

Would it make sense to split the docker publish and heroku deploy jobs? I agree that docker releases make more sense to be aligned with actual Companion releases. But the heroku deploy can use the main branch of Companion, just like examples on the website use the main branch of Uppy.